### ISSUE TITLE
error during make

### ISSUE THREAD

**Issue by flip111** [seliopou/elm-d3#18]
System: ubuntu 15.04
ELM_HOME: /usr/lib/node_modules/elm/share
node version: v4.0.0
npm version: 2.14.2
elm version: don't know, installed with https://www.npmjs.com/package/elm

commands:
```
git clone --recursive --single-branch --branch master https://github.com/seliopou/elm-d3.git
cd elm-d3
npm install smash
make
```

```
» make                                                              [15:07:14]
./node_modules/.bin/smash src/Native/runtime.js > elm-d3.runtime.js
elm --src-dir=src --make -o src/D3.elm
Could not find command `--src-dir=src`. Maybe there is a typo?

Default commands include:

  make      Compile an Elm file or project into JS or HTML
  package   Manage packages from <http://package.elm-lang.org>
  reactor   Develop with compile-on-refresh and time-travel debugging
  repl      A REPL for running individual expressions

When you try to run the command `--src-dir=src` we actually search for an
  executable named elm---src-dir=src. Are you able to run elm---src-dir=src?
  Is it on your PATH?
elm --src-dir=src --make -o src/D3/Color.elm
Could not find command `--src-dir=src`. Maybe there is a typo?

Default commands include:

  make      Compile an Elm file or project into JS or HTML
  package   Manage packages from <http://package.elm-lang.org>
  reactor   Develop with compile-on-refresh and time-travel debugging
  repl      A REPL for running individual expressions

When you try to run the command `--src-dir=src` we actually search for an
  executable named elm---src-dir=src. Are you able to run elm---src-dir=src?
  Is it on your PATH?
elm --src-dir=src --make -o src/D3/Event.elm
Could not find command `--src-dir=src`. Maybe there is a typo?

Default commands include:

  make      Compile an Elm file or project into JS or HTML
  package   Manage packages from <http://package.elm-lang.org>
  reactor   Develop with compile-on-refresh and time-travel debugging
  repl      A REPL for running individual expressions

When you try to run the command `--src-dir=src` we actually search for an
  executable named elm---src-dir=src. Are you able to run elm---src-dir=src?
  Is it on your PATH?
elm --src-dir=src --make -o src/D3/Scale.elm
Could not find command `--src-dir=src`. Maybe there is a typo?

Default commands include:

  make      Compile an Elm file or project into JS or HTML
  package   Manage packages from <http://package.elm-lang.org>
  reactor   Develop with compile-on-refresh and time-travel debugging
  repl      A REPL for running individual expressions

When you try to run the command `--src-dir=src` we actually search for an
  executable named elm---src-dir=src. Are you able to run elm---src-dir=src?
  Is it on your PATH?
elm --src-dir=src --make -o src/D3/Voronoi.elm
Could not find command `--src-dir=src`. Maybe there is a typo?

Default commands include:

  make      Compile an Elm file or project into JS or HTML
  package   Manage packages from <http://package.elm-lang.org>
  reactor   Develop with compile-on-refresh and time-travel debugging
  repl      A REPL for running individual expressions

When you try to run the command `--src-dir=src` we actually search for an
  executable named elm---src-dir=src. Are you able to run elm---src-dir=src?
  Is it on your PATH?
./node_modules/.bin/smash build/src/D3.js build/src/D3/Color.js build/src/D3/Event.js build/src/D3/Scale.js build/src/D3/Voronoi.js > elm-d3.library.js
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open 'build/src/D3.js'
    at Error (native)
Makefile:25: recipe for target 'elm-d3.library.js' failed
make: *** [elm-d3.library.js] Error 1
```

**Comment by seliopou** [comment]
This almost certainly due to a new release of the Elm compiler. The library as-is expects elm-0.13, and may require some minor tweaks to get working. To confirm, can you run `elm --version` and report back the output?

**Comment by flip111** [comment]
sure thing!

```
» elm --version                                                     [21:37:08]
Could not find command `--version`. Maybe there is a typo?

Default commands include:

  make      Compile an Elm file or project into JS or HTML
  package   Manage packages from <http://package.elm-lang.org>
  reactor   Develop with compile-on-refresh and time-travel debugging
  repl      A REPL for running individual expressions

When you try to run the command `--version` we actually search for an
  executable named elm---version. Are you able to run elm---version?
  Is it on your PATH?
```

**Comment by jarandaf** [comment]
Report `elm` command output, first line corresponds to the platform version.

**Comment by flip111** [comment]
`Elm Platform 0.15.1 - a way to run all Elm tools`

**Comment by seliopou** [comment]
Watch the [compat](https://github.com/seliopou/elm-d3/tree/compat) branch for a resolution of this issue.

@flip111 for your purposes you may also be interested in [ocaml-d3](https://github.com/seliopou/ocaml-d3).

**Comment by seliopou** [comment]
Fixed with 7ccddd847925a948e8f5bde444c8836a976e5058
